docs/geedocs/5.2.4/answer/7160004.html

```diff
@@ -79,14 +97,14 @@ <h5>
             </td>
           </tr>
           <tr>
-            <td>881</td>
-            <td>Error Saving Map Layer with Icon</td>
+            <td>946</td>
+            <td>Panoramic Streetview JS error</td>
```

Comment:
Expand "JS" to "JavaScript", or if the errors only appear in the JS console, the use "debug console"